core/src/maker.ts

```diff
@@ -26,6 +27,7 @@ const createMaker = async function (network: string): Promise<typeof Maker> {
             url: networkConfig.url,
             type: 'HTTP',
         },
+        privateKey,
```

Comment:
As I suggested, and you pointed out in the comment above, this may not work and before debugging our code, please validate that passing `privateKey` is actually properly handled by the dai.js library. You may find this in the dai.js docs, in the code itself or in the test. It may be that you also need to pass something else or at least call `maker.useAccountWithAddress(address)`

Reply:
I had assumed as you sent me the following code snippet, but I will look further into it!